src/composables/useTipMessage.js

```diff
@@ -0,0 +1,18 @@
+import showTipMessage from '#src/graphql/mutation/tipMessage/showTipMessage.graphql';
+
+export default function useTipMessage(apollo) {
+	const $showTipMsg = (tipMsg, tipMsgType = '', tipPersist = false) => {
+		apollo.mutate({
```

Comment:
We probably want a `.catch` here.

Reply:
Current mixin doesn't have but a good call to add it.